src/tests/test_policies.cpp

```diff
@@ -22,6 +22,8 @@ version: "1.0"
       input_type: mock
       config:
         iface: eth0
+  global_handler_config:
```

Comment:
You're right sorry, but it does need to be in rfc and it should still be broken out per handler.

Reply:
Oh i guess we don't have an rfc for config! We should fix that too :P

Reply:
yes right now you have a section which will apply key/val config to _all_ handlers but i want to be able to specify global key/val config per handler. so instead of

```
visor:
  taps:
    anycast:
      input_type: mock
      config:
        iface: eth0
  global_handler_config:
    only_rcode: 2
```
it should be 

```
visor:
  taps:
    anycast:
      input_type: mock
      config:
        iface: eth0
  global_handler_config:
    dns:
      only_rcode: 2
```

Reply:
done. We should discuss better how to add a RFC for this and also for Config. It should be two separated RFCs? 
if so, what should be the name of them?

Reply:
no just a single RFC for config file schema. it should follow the pattern of the other RFCS - create an issue for it, then create the RFC file name with the data and issue number in it.

we can land this though and do that separately.

Reply:
issue created: https://github.com/ns1labs/pktvisor/issues/307

I will land this one